config/base/tekton-config-defaults.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tekton-config-defaults
+  labels:
+    operator.tekton.dev/release: devel
+
+data:
+  AUTOINSTALL_COMPONENTS: "true"
+  DEFAULT_NAMESPACE: ""
```

Comment:
ux suggestion. it will be helpful for users, if we name it `DEFAULT_TARGET_NAMESPACE` . 
otherwise, our users might think this is the default namespace where the operator gets installed, (instead of thinking about the components installation)